swagger/public_v1/component_schemas/ProviderFilter.yml

```diff
@@ -2,6 +2,39 @@ type: object
 example: ""
 description: "This schema is used to search within collections to return more specific results."
 properties:
+  can_sponsor_skilled_worker_visa:
+    description: "Only return providers that can sponsor skilled worker visa."
+    type: boolean
+    example: true
+  can_sponsor_student_visa:
+    description: "Only return providers that can sponsor student visa."
+    type: boolean
+    example: true
+  provider_type:
+    description: "Return providers based on their provider type. This is a comma delimited string. If multiple provider types are provided then any provider matching any one of the options provider will be returned, i.e. the OR operator is used."
+    type: string
+    example: "scitt,lead_school"
+    enum:
+      - scitt
+      - lead_school
+      - university
+  region_code:
+    description: "Return providers based on their region code. This is a comma delimited string. If multiple region codes are provided then any provider matching any one of the options provider will be returned, i.e. the OR operator is used."
+    type: string
+    example: "london,yorkshire_and_the_humber"
+    enum:
+      - no_region
+      - london
+      - south_east
+      - south_west
+      - wales
+      - west_midlands
+      - east_midlands
+      - eastern
+      - north_west
+      - yorkshire_and_the_humber
+      - north_east
+      - scotland
```

Comment:
```suggestion
      - no_region
      - north_east
      - north_west
      - yorkshire_and_the_humber
      - east_midlands
      - west_midlands
      - eastern
      - london
      - south_east
      - south_west
      - scotland
      - wales
```

Reply:
Done in https://github.com/DFE-Digital/publish-teacher-training/pull/3150/commits/091499a197ec6cc6e8daae6fe174585c28b95f20